src/geolocation.js

```diff
@@ -0,0 +1,33 @@
+import Geolocation from "@react-native-community/geolocation";
```

Comment:
Can we put this file in some API folder in order to structure it perfectly ?

Reply:
This would be useful. It's not uncommon to have a separate API folder.

Reply:
I've added the `/api` folder and moved `geolocation.js` inside it.